biblib/views/library_view.py

```diff
@@ -337,9 +344,14 @@ def get(self, library):
 
         sort = request.args.get('sort', 'date desc')
         #timestamp sorting is handled in biblib so we need to change the sort to something SOLR understands.
-        if "time" in sort:
-            add_sort = sort 
+        if sort in ['time asc', 'time desc']:
+            current_app.logger.debug("sort order is set to{}".format(sort))
+            if 'time desc'== sort:
```

Comment:
Very minor, but usually this equality is flipped (e.g. `if sort == "time desc"`)